guide/getting-started/software.md

```diff
@@ -169,8 +169,12 @@ A [node]({{ '/guide/glossary/node/' | relative_url }}) is simply a computer runn
    layout = "float-right-desktop"
 %}
 
+Bitcoin mining is the act of confirming the next block in the blockchain by being the first to solve a computationally intense math problem. The successful miner receives a block reward and the fees for the transactions contained in the new block in bitcoin.
```

Comment:
```suggestion
Bitcoin [mining]({{ '/guide/getting-started/technology-primer/#how-are-transactions-confirmed' | relative_url }}) is the act of confirming transactions by solving a computationally intense problem, which is rewarded via freshly minted bitcoin and transaction fees.
```
Wouldn't let me make a suggestion on the previous thread, so posting it as a new comment. How about shortening it and linking to the tech primer (as @Bosch-0 suggested)?

Reply:
Will update in a new commit as this became outdated when I removed that paragraph.